addon/controllers/scopes/edit.js

```diff
@@ -1,36 +1,62 @@
 import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
+import { localCopy } from "tracked-toolbox";
 
 import PaginationController from "ember-emeis/-private/controllers/pagination";
 
 export default class ScopesEditIndexController extends PaginationController {
   @service emeisOptions;
+  @service notification;
   @service intl;
   @service router;
   @service store;
 
+  @localCopy("model.parent") parent;
+
   get queryParamsfilter() {
-    return { scope: this.model?.id };
+    return { scope: this.model?.id ?? "" };
   }
 
   get metaFields() {
     return this.emeisOptions.scope?.metaFields;
   }
 
-  get allScopes() {
-    return this.store.peekAll("scope");
+  get childScopeIds() {
+    return this.model?.findChildren().map((s) => s.id) ?? [];
+  }
+
+  get allOtherScopes() {
+    return this.store
+      .peekAll("scope")
+      .filter(
+        (scope) =>
+          scope.id !== this.model?.id && !this.childScopeIds.includes(scope.id)
+      );
   }
 
   @action
   updateModel(model, formElements) {
     model.name = formElements.name.value;
     model.description = formElements.description.value;
+    if (
+      this.parent
+        ?.findParents()
+        .map((scope) => scope.id)
+        .includes(this.model.id)
+    ) {
+      this.parent = model.parent;
+      this.notification.warning(
+        this.intl.t("emeis.form.invalid-scope-assignment-warning")
+      );
+    } else {
+      model.parent = this.parent;
+    }
```

Comment:
can we drop this since we don't offer child scopes for selection anymore?

Reply:
yep!